tests/ui/perfherder/graphs_view_test.jsx

```diff
@@ -226,11 +227,47 @@ test('Using select query param displays tooltip for correct datapoint', async ()
   const revision = await waitForElement(() =>
     getByText(expectedRevision.slice(0, 13)),
   );
-
+  const repoName = await waitForElement(() => getByTestId('repoName'));
   expect(graphTooltip).toBeInTheDocument();
   expect(revision).toBeInTheDocument();
+  expect(repoName).toHaveTextContent(testData[0].repository_name);
```

Comment:
The expected values aren't identified & checked in a consistent manner.
`expectedRevision` duplicated the value to check *(from testData)*, while `repoName` is compared in a more dynamic way, directly with the value from testData.

We should stick to a single approach, so the code is more readable.

Reply:
I see that in the whole code the first method has been used but in my opinion, the dynamic way is better as it checks everything exactly as it is.
WDYT should I use @ionutgoldan

Reply:
Dynamic use is fine.

Reply:
> [...]
> WDYT should I use @ionutgoldan

My answer is late & no longer relevant. But I don't have a preference.

Reply:
> > [...]
> > WDYT should I use @ionutgoldan
> 
> My answer is late & no longer relevant. But I don't have a preference.

Not at all. Your opinion matters me a lot.
I have learnt from you a lot :)